<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QMainWindow" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>796</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>client</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>971</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { border: 2px solid #C4C4C3; }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="admin">
     <attribute name="title">
      <string>Admin</string>
     </attribute>
     <widget class="QPushButton" name="pbn_view_database_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>460</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>View Database</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_view_database_error_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Data Base</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/Bol7assan/Downloads/New folder/1641115343650.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tbl_view_database_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>941</width>
        <height>141</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>AccountNumber</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Full Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Balance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Age</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="lbl_err_transaction_history_2">
      <property name="geometry">
       <rect>
        <x>421</x>
        <y>435</y>
        <width>16</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>16</y>
        <width>691</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QPushButton" name="pushButton_get_account_number_2">
           <property name="text">
            <string>Get Account Number</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_username_2">
             <property name="text">
              <string>username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_username_2"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_view_account_number_2">
           <property name="text">
            <string>Account Number:               </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_error_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="pbn_view_balance_2">
           <property name="text">
            <string>View Balance</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_20">
           <item>
            <widget class="QLabel" name="lbl_accountnumber_2">
             <property name="text">
              <string>Account Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnedit_accountnumber_2"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lbl_viewbalance_2">
           <property name="text">
            <string>Balance:                                 </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_error_viewbalance_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>170</y>
        <width>941</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lbl_accountcreation_2">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Account Creation</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_21">
             <item>
              <widget class="QLabel" name="lbl_username_input_2">
               <property name="text">
                <string>Username: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lnedit_username_input_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_22">
             <item>
              <widget class="QLabel" name="lbl_password_2">
               <property name="text">
                <string>Password: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lnedit_password_input_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lbl_name_2">
               <property name="text">
                <string>Name: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lnedit_name_input_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_24">
             <item>
              <widget class="QLabel" name="lbl_age_2">
               <property name="text">
                <string>Age: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lnedit_age_input_2"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="pbn_create_new_account_2">
               <property name="text">
                <string>Create New Account</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkbox_admin_2">
               <property name="text">
                <string>Admin</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lbl_error_create_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Account Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnedit_act_number_transaction_history_2"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbn_view_transaction_history_2">
             <property name="text">
              <string>View Transaction history</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tbl_transaction_history_2">
           <column>
            <property name="text">
             <string>Transaction ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Amount</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Date</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Time</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="user">
     <attribute name="title">
      <string>User</string>
     </attribute>
     <widget class="QLabel" name="lbl_trnsfr_amnt_2">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>20</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/Bol7assan/Downloads/New folder/download.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>346</y>
        <width>931</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="lbl_view_trnsction_history_2">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>View Transaction History</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>                      </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_err_transaction_history_3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbn_view_transaction_histroy_2">
           <property name="text">
            <string>View Transaction History</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tbl_view_histroy_transaction_2">
         <column>
          <property name="text">
           <string>Transaction ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Amount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>574</width>
        <height>318</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_account_number_2">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_get_account_number_4">
             <property name="text">
              <string>Get Account Number</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_view_balance_2">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbn_view_balance_4">
             <property name="text">
              <string>View Account Balance</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="lbl_title_trasnaction_2">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>18</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Make a Transaction</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lbl_amount_2">
               <property name="text">
                <string>Amount: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lnedit_amount_2">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="lbl_transaction_error_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbn_make_trasnaction_2">
             <property name="text">
              <string>Make a Transaction</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Times New Roman</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Make Transfer</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="lbl_to_acc_no_2">
             <property name="text">
              <string>To Account Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnedit_to_accountnumber_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="lbl_transfe_amout_2">
             <property name="text">
              <string>Transfer Amount:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lnedit_trnsfr_amount_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="lbl_mk_trnsf_err_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbn_mk_transfer_2">
           <property name="text">
            <string>Make a Transfer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>378</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/Bol7assan/Downloads/New folder/1641115343650.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_Username">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Username"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_Password">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_Password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_login">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_logout">
         <property name="text">
          <string>Log out</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1232</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
